VAMobile/src/components/NotificationManager/NotificationManager.tsx

```diff
@@ -1,48 +1,65 @@
-import React, { FC, useEffect, useState } from 'react'
+import React, { Dispatch, FC, SetStateAction, createContext, useContext, useEffect, useState } from 'react'
 import { Linking, View } from 'react-native'
 import { NotificationBackgroundFetchResult, Notifications } from 'react-native-notifications'
 import { useSelector } from 'react-redux'
 
+import { useRegisterDevice } from 'api/notifications'
 import { usePersonalInformation } from 'api/personalInformation/getPersonalInformation'
 import { Events } from 'constants/analytics'
 import { RootState } from 'store'
 import { AuthState } from 'store/slices'
-import {
-  dispatchSetInitialUrl,
-  dispatchSetTappedForegroundNotification,
-  registerDevice,
-} from 'store/slices/notificationSlice'
 import { logAnalyticsEvent } from 'utils/analytics'
-import { useAppDispatch } from 'utils/hooks'
 
 const foregroundNotifications: Array<string> = []
 
+interface NotificationContextType {
+  tappedForegroundNotification: boolean
+  initialUrl: string
+  setTappedForegroundNotification: Dispatch<SetStateAction<boolean>>
+  setInitialUrl: Dispatch<SetStateAction<string>>
+}
+
+export const NotificationContext = createContext<NotificationContextType>({
```

Comment:
```suggestion
const NotificationContext = createContext<NotificationContextType>({
```
Since this isn't imported anywhere else

Reply:
updated